php/libraries/Visit.class.inc

```diff
@@ -24,13 +24,6 @@ namespace LORIS;
  */
 class Visit
 {
-    /**
-     * The visit ID
-     *
-     * @var ?string
-     */
-    protected $id;
```

Comment:
This is a protected variable which has no getter and isn't used internally in the class.